### PR TITLE
[LP-8] Update navbar

### PR DESCRIPTION
**Pull request by IvanYarochkin**

### CODE REVIEW

File: src/main/webapp/app/layouts/navbar/navbar.component.html [hunk 1]
```diff
@@ -17,7 +17,7 @@
                 </a>
             </li>
             <!-- jhipster-needle-add-element-to-menu - JHipster will add new menu items here -->
-            <li *ngSwitchCase="true" ngbDropdown class="nav-item dropdown pointer" display="dynamic" routerLinkActive="active" [routerLinkActiveOptions]="{exact: true}">
```
Comment (xxMikhailx) [inline src/main/webapp/app/layouts/navbar/navbar.component.html:20]: Why did you delete this `*ngSwitchCase="true"`?

Comment (IvanYarochkin) [reply src/main/webapp/app/layouts/navbar/navbar.component.html:20]: Since *ngSwitchCase="true" hides the content for unregistered users.

Comment (xxMikhailx) [reply src/main/webapp/app/layouts/navbar/navbar.component.html:20]: Ok

**Comment by xxMikhailx** [comment]
For this case it's ok but please name your next branches only using lowercase.

**Review by xxMikhailx — DISMISSED** [review]

**Review by xxMikhailx — APPROVED** [review]
